templates/example_neighborhood_analysis_script.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>We should add a parameter that controls whether the value is overlaid over the heatmap or not, and set the default to False</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/358/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>